doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1054,16 +931,9 @@ described first.
    the usual defined constants, whose actual values are of course
    relevant in general.
 
-   .. cmdv:: Global Transparent {+ @qualid }
-      :name: Global Transparent
+   .. exn:: The reference @smart_qualid was not found in the current environment.
```

Comment:
This error message also appeared in the `Transparent` command and should have been fixed in the same way. There's no point in repeating it (it's the same error), so I would remove the one in `Opaque`.

Reply:
Should the message text have @smart_qualid or @qualid?

Reply:
In fact, it should be `qualid` because the error message is not the same if you provide a `string` that could not be associated to a `qualid`. In this case, the error message is "`Unable to interpret @string as a reference.`"

That being said, these two error messages are not specific to `Opaque` and `Transparent` and ideally, they should be documented only once, in the documentation of `smart_qualid`. For now, it is defined in the grammar of `Arguments`, but eventually it should be moved and properly explained in the "Syntax extensions" chapter.

Reply:
I left the message here for now.